src/components/ServerList.tsx

```
@@ -126,1062 +109,896 @@ const themeLightWarm = themeQuartz.withPart(colorSchemeLightWarm);
+              <br />
+              <h1 className="animate-fade-in -translate-y-4 text-balance bg-gradient-to-br from-black from-30% to-black/40 bg-clip-text pb-6 text-2xl font-semibold leading-none tracking-tighter text-transparent opacity-0 [--animation-delay:200ms] sm:text-2xl md:text-3xl lg:text-4xl dark:from-white dark:to-white/40">
+                Make your server stand out
+              </h1>
+              <p className="animate-fade-in mb-12 -translate-y-4 text-balance text-lg tracking-tight text-gray-400 opacity-0 [--animation-delay:400ms] md:text-xl">
+                Servers can have custom banners, Discord widgets, color schemes,
+                and descriptions, making your server stand out with information
+                that can be shown to players.
+              </p>
+              <BentoGrid className="max-h-[100px] mb-[300px]">
+                {features.map((feature, idx) => (
+                  <BentoCard key={idx} {...feature} />
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)